### ISSUE TITLE
Rubocop style violations

### ISSUE THREAD

**Issue by soumyaray** [Lemon555/NEO#5]
Please run rubocop on all your code:  you've got strange indentation issues!

**Comment by marandy0905** [comment]
@soumyaray Yes... we have quite a problem there and heard about the `rubocop.yml` and we'll check for that!

**Comment by soumyaray** [comment]
What problem did you have?  No need for rubocop.yml unless you want to specify the version of rubocop you want to use.  Please don't use it to deactivate any cops :D
